src/lib/lang/evaluators/order1/parenExpression.ts

```diff
@@ -0,0 +1,27 @@
+import type { TreeCursor } from '@lezer/common'
+import { Sequence } from '../../dataStructures'
+import { evalOrderLast } from ".."
+import { ErrorNodeError } from '../errors'
+
+    //for the paren expression we just need to go through everything within the parentheses and evaluate it FIRST before evaluating other things
+export function evalParenExpression(src: string, node: TreeCursor): number | Sequence{
+
+    // result can either evaluate to a flat number or be a sequence depending on the contents within the parentheses
+    let results: number|Sequence;
```

Comment:
Nitpick: Can we move this down closer to where it's used? That way it spends a minimal amount of time taking up space in memory.

Reply:
nitpick fixed